docs/hydrostatic-pressure/index.md

```diff
@@ -72,6 +72,9 @@ finally:
 - Line 56 prints the results
 - In the end 58-60 closes the connection with the database
 
+This code can be accessed at
```

Comment:
Replace lines 75-76 to `[Download the code](https://github.com/TOKU-Systems/tutorials/blob/develop/docs/hydrostatic-pressure/hydrostatic_pressure.py)`